packages/core-components/src/layout/Page/PageWithHeader.tsx

```diff
@@ -16,25 +16,20 @@
 
 import React from 'react';
 
-import { Header, Page } from '@backstage/core-components';
-import { useApi, configApiRef } from '@backstage/core-plugin-api';
+import { Header } from '../Header';
+import { Page } from './';
 
-type Props = {
-  children?: React.ReactNode;
-};
-export const ApiExplorerLayout = ({ children }: Props) => {
-  const configApi = useApi(configApiRef);
-  const generatedSubtitle = `${
-    configApi.getOptionalString('organization.name') ?? 'Backstage'
-  } API Explorer`;
-  return (
-    <Page themeId="apis">
-      <Header
-        title="APIs"
-        subtitle={generatedSubtitle}
-        pageTitleOverride="APIs"
-      />
-      {children}
-    </Page>
-  );
-};
+interface IProps extends React.ComponentProps<typeof Header> {
```

Comment:
Thanks for adding the article to your comment - really helpful & definitely smth. I will keep in mind now!! :)

I used it, because it was a pattern at a company I worked before - thanks for questioning it! :)